app/main.ipynb

```diff
@@ -216,15 +282,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A.mean(axis=0)\n",
```

Comment:
In Task 5.2, the code calculates the mean instead of the standard deviation. Use `np.std(A, axis=0)` to get the standard deviation by the y-axis.